### ISSUE TITLE
bigquery: After upgrade v 1.11.0, bigquery.RowIterator pager return empty result.

### ISSUE THREAD

**Issue by hue-plu** [googleapis/google-cloud-go#2946]
**Client**
BigQuery

**Go Environment**

$ go version
go version go1.13.6 darwin/amd64

**Code**

```go
package main

import (
	"context"
	"fmt"

	"cloud.google.com/go/bigquery"
	"google.golang.org/api/iterator"
)

func main() {
	ctx := context.Background()
	client, _ := bigquery.NewClient(ctx, projectId)
	defer client.Close()

	query := `
          SELECT
            year, name
          FROM
            bigquery-public-data.usa_names.usa_1910_2013
          LIMIT 5;
        `
	result := client.Query(query)
	iter, err := result.Read(ctx)
	if err != nil {
		panic(err.Error())
	}

	pager := iterator.NewPager(iter, 2, "")
	nextPageToken := ""
	for {
		var rows [][]bigquery.Value
		nextPageToken, err = pager.NextPage(&rows)
		fmt.Printf("totalRows: %v, rows: %+v\n", iter.TotalRows, rows)

		if err != nil {
			panic(err.Error())
		}

		if nextPageToken == "" {
			break
		}
	}
}
```

**Expected behavior**

When cloud.google.com/go/bigquery version set to v1.10.0
`pager.NextPage` return TotalRows and Query Results like below.

```sh
$ go run main.go
totalRows: 5, rows: [[1910 Elsie] [1910 Annie]]
totalRows: 5, rows: [[1910 Katie] [1910 Alma]]
totalRows: 5, rows: [[1910 Thelma]]
```

**Actual behavior**

But when I set cloud.google.com/go/bigquery version to v1.11.0
pager.NextPage return empty result.

```sh
$ go run main.go
totalRows: 0, rows: []
```

**Comment by codyoss** [comment]
@hue-plu Does upgrading to v1.11.1 fix the issue?

**Comment by shollyman** [comment]
Baking a release with this now.

**Comment by shollyman** [comment]
Released as [bigquery/v1.11.2](https://github.com/googleapis/google-cloud-go/releases/tag/bigquery%2Fv1.11.2)

**Comment by hue-plu** [comment]
Thank you for your quick response and fix!
